ugv:
  cmd_manager_srv: "/robot/command_manager/command"
  cmd_sequencer_srv: "/robot/command_sequencer/command"
  move_to_action: "/robot/arm/move_to"
  pickup_from_action: "/robot/arm/pickup_from"
  place_on_action: "/robot/arm/place_on"
  dock_action: "/robot/base/dock"
  odom_topic: "/robot/odom"
  find_offset_srv: "/robot/find_offset"
  get_deposit_srv: "/robot/get_deposit_sequence"
  body_cam_ns: "/robot/body_camera/front_rgbd_camera"
  arm_cam_ns: "/robot/arm_camera/front_rgbd_camera"

iccs:
  find_pothole_srv: "/iccs/find_pothole"

pothole:
  inspection_names: 
    - "inspection_mid"
    - "inspection_left"
    - "inspection_right"
  roller_sequence:
    - "POT_FB_1"
    - "POT_FB_2"
    - "POT_LR_1"
    - "POT_LR_2"

cone_place:
  gps_points:
    - [37.7745, -122.4195]  # P1
    - [37.7745, -122.4185]  # P2
    - [37.7740, -122.4197]  # P3
    - [37.7740, -122.4190]  # P4
    - [37.7740, -122.4183]  # P5
